From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Sun, 28 Apr 2024 11:46:31 +0200
Subject: [PATCH 1/1] Fix building on case sensitive mingw platforms

Mingw headers are all lowercase, and can be used for cross compilation
from case sensitive file systems.

The official Windows SDK headers aren't self-consistent wrt upper/lower
case, so those headers can't be used on case sensitive systems without
a layer providing case insensitivity anyway.

Upstream-Status: Pending

diff --git a/src/io.c b/src/io.c
index 1111111..2222222 100644
--- a/src/io.c
+++ b/src/io.c
@@ -32,7 +32,7 @@
 #if defined(_WIN32)
 #define WIN32_LEAN_AND_MEAN
 #define NOGDI
-#include <Windows.h>
+#include <windows.h>
 #endif
 #include <stdlib.h>
 #include <stdarg.h>
